tests/state/test_state.py

```diff
@@ -37,13 +35,13 @@ def test_config(self):
 
     def test_memoize(self):
         @state.memoize(0.1)
-        def rand():
+        def rand() -> float:
             return random.random()
 
         assert rand() == rand()
         rand1 = rand()
         assert rand1 == rand()
-        time.sleep(0.1)
+        time.sleep(1)
```

Comment:
Why this change?

Reply:
Oops... I thought about changing this since `memoize` is actually supposed to take an int not a float, but I'm going to switch this back since I think it's better to avoid making the tests slower.